docs/helpers.rst

```diff
@@ -175,23 +175,28 @@ The user model used by Django. This handles different versions of Django.
 
 The field name used for the username on the user model.
 
+.. fixture:: db
+
+
 ``db``
 ~~~~~~~
 
-.. fixture:: db
-
 This fixture will ensure the Django database is set up.  This only
 required for fixtures which want to use the database themselves.  A
-test function should normally use the :py:func:`~pytest.mark.django_db`
+test function should normally use the :py:func:`pytest.mark.django_db`
 mark to signal it needs the database.
 
+.. fixture:: transactional_db
+
 ``transactional_db``
```

Comment:
Ah, good point of moving the `.. fixture:` anchor above.
Should be done also for the other existing ones I've added inbetween, and then maybe add it for all fixtures (I only did it for new ones / when editing others).

Reply:
Hi, 

thank you for taking time to re-review. 

I don't understand "LGTM" acronym. 

Also, per your comment "good point of moving the .. fixture: anchor" and the rest of it, it is unclear to me if you need me to perform any actions on the file or not.

Reply:
LGTM = Looks goot to me.

You could also move the `.. fixture …` for other fixtures, but you do not have to.  Would be good though, of course.